sys/test_utils/result_output/result_output.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys
+ * @{
+ *
+ * @file
+ * @brief       Result output implementation
+ *
+ *
+ * @author      Kevin Weiss <kevin.weiss@haw-hamburg.de>
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "assert.h"
+#include "fmt.h"
+
+#include "test_utils/result_output.h"
+
+void turo_data_int(int turo, int32_t data)
+{
+    (void)turo;
+    print_str("{\"data\": ");
```

Comment:
should we add a space here and there, so it looks a bit more pretty printed? e.g., here after "}"

Reply:
we can do that in a folo

Reply:
Ok, I will push after mrudock is done.

Reply:
Done